### PR TITLE
Updates to why and how to get involved

### LINKED ISSUE #144: Adjust Documentation
Right before launching, we will need to: 

- [x] Adjust the language of the README after #143 
- [x] Move all the Hugo administration how-to documentation from the README.md to a better place (TBD)

Might be resolved with #90 but possibly separate.

### PR DESCRIPTION
**Pull request by mbbroberg**
As we get closer to launch, want to give more details on what we're up to!

Fixes #144

### CODE REVIEW

**Comment by cwjohnston** [comment]
I'm not sure why this PR is in this state where we're only adding a single line. 

@mbbroberg will you please take a look at the current README in master and determine if the changes intended in this PR are already present in master?

**Comment by mbbroberg** [comment]
@cwjohnston fixed it. This will need a fresh review.

File: README.md [hunk 1]
```diff
@@ -45,6 +47,14 @@ cd sensu-docs-site ; git submodule update --init --recursive
 This project uses [Yarn](https://yarnpkg.com/) to manage dependencies and the build process. For installation
 on installing yarn, [view their documentation](https://yarnpkg.com/lang/en/docs/install/).
 
+#### Installing Hugo
```
Comment (cwjohnston) [inline README.md]: Installing `hugo` is now handled automatically when running `yarn` or `yarn server`, so we shouldn't need these instructions any longer.

**Comment by mbbroberg** [comment]
@cwjohnston got it 👍

**Pull request merged**